app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -20,6 +20,18 @@ def self.event_monitor_class
     ManageIQ::Providers::Openshift::ContainerManager::EventCatcher
   end
 
+  def common_logging_route_name
+    Settings.container_logging.common_logging_route
+  end
+
+  def common_logging_query
+    nil # should be empty to return all
+  end
+
+  def common_logging_path
+    '/#/dashboard/Operations-Logs-Overivew'
```

Comment:
@enoodle is this dashboard always available? Because IIUC it's not shipped with Kibana.